src/components/Datepicker/Datepicker.vue

```diff
@@ -0,0 +1,291 @@
+      :is-open="isOverlayVisible"
+      :reference="inputElement"
+      placement="bottom-start"
+      class="sb-datepicker__overlay"
+    >
+      <SbDatepickerHeader
+        v-bind="{
+          isCalendarView,
+          isYearView,
+          isMonthView,
+          isTimeView,
+          value: internalDate,
+        }"
+        @previous-month="handlePreviousMonth"
+        @next-month="handleNextMonth"
+        @change-month="handleChangeMonth"
+        @change-year="handleChangeYear"
+      />
+
+      <component
+        :is="isComponentView"
+        :value="internalValue"
+        :internal-date="internalDate"
+        @input="handleComponentsInput"
+      />
+
+      <div class="sb-datepicker__actions">
+        <button
+          class="sb-datepicker__action-button"
+          @click="handleCancelAction"
+        >
+          Cancel
+        </button>
+
+        <button
+          class="sb-datepicker__action-button sb-datepicker__action-button--primary"
+          @click="handleDoneAction"
+        >
+          Done
+        </button>
+      </div>
+    </SbPopover>
+  </div>
+</template>
+
+<script>
+import dayjs from 'dayjs'
+
+import { ClickOutside } from '../../directives'
+import { includes } from '../../utils'
+import { SbTextField } from '../TextField'
+import { SbPopover } from '../Popover'
+import SbTooltip from '../Tooltip'
+
+import SbDatepickerHeader from './components/DatepickerHeader'
+import SbDatepickerTime from './components/DatepickerTime'
+import SbDatepickerDays from './components/DatepickerDays'
+import SbDatepickerMonths from './components/DatepickerMonths'
+import SbDatepickerYears from './components/DatepickerYears'
+
+import { datepickerOptions, INTERNAL_VIEWS } from './utils'
+
+export default {
+  name: 'SbDatepicker',
+
+  components: {
+    SbTooltip,
+    SbPopover,
+    SbTextField,
+    SbDatepickerHeader,
+    SbDatepickerDays,
+    SbDatepickerTime,
+    SbDatepickerMonths,
+    SbDatepickerYears,
+  },
+
+  directives: {
+    ClickOutside,
+  },
+
+  props: {
+    disabled: Boolean,
+
+    placeholder: {
+      type: String,
+      default: null,
+    },
+
+    type: {
+      type: String,
+      default: 'datetime',
+      validations: (val) => includes(datepickerOptions, val),
+    },
+
+    tzOffset: {
+      type: String,
+      default: '',
+    },
+
+    tzTooltip: {
+      type: String,
+      default: null,
+    },
+
+    value: {
+      type: String,
+      default: '',
+      required: true,
+    },
+  },
+
+  data: () => ({
+    forceVisibleTime: false,
+    internalDate: dayjs().format(),
+    internalValue: dayjs().format(),
+    inputElement: null,
+    isOverlayVisible: false,
+    internalVisualization: INTERNAL_VIEWS.CALENDAR,
+  }),
+
+  computed: {
+    internalFormat() {
+      const FORMATS = {
+        time: 'HH:mm',
+        date: 'YYYY-MM-DD',
+        datetime: 'YYYY-MM-DD HH:mm',
+      }
+
+      return FORMATS[this.type]
+    },
+
+    internalValueFormated() {
```

Comment:
if I'm not mistaken, it should be "Formatted" instead of "Formated"

Reply:
You're right, thanks